src/main/java/seedu/parser/MainParser.java

```diff
@@ -84,12 +91,17 @@ private Command parseEditContact(String userInput) { // userInput is raw user in
         try {
             // split into array of size 3 with command, index and details
             String[] details = userInput.split(" ", NUMBER_OF_EDIT_DETAILS);
+            if (details[ISOLATE_USER_INPUT].trim().equalsIgnoreCase("")) {
+                throw new MissingDetailException();
+            }
```

Comment:
Hi @ng-andre , I believe your intention is to check for user inputs like `edit 1` without any specified edits. In that case, as per @lezongmun 's suggestion, a `MissingArgException` would be more suitable here.
FYI: `MissingArgException` is for commands that do not have any specified arguments, while `MissingDetailException` is for commands that specify a flag, but do not specify any argument **for that flag**.